pycode/memilio-epidata/memilio/epidata/getDIVIData.py

```diff
@@ -168,23 +218,94 @@ def get_divi_data(read_data=dd.defaultDict['read_data'],
     df_states.reset_index(inplace=True)
     df_states.sort_index(axis=1, inplace=True)
 
-    filename = "state_divi"
-    filename = gd.append_filename(filename, impute_dates, moving_average)
-    gd.write_dataframe(df_states, directory, filename, file_format)
-
     # write data for germany to file
     df_ger = df.groupby(["Date"]).agg({"ICU": "sum", "ICU_ventilated": "sum"})
     df_ger.reset_index(inplace=True)
     df_ger.sort_index(axis=1, inplace=True)
 
-    filename = "germany_divi"
-    filename = gd.append_filename(filename, impute_dates, moving_average)
-    gd.write_dataframe(df_ger, directory, filename, file_format)
+    if not to_dataset:
+        filename = "county_divi"
+        filename = gd.append_filename(filename, impute_dates, moving_average)
+        gd.write_dataframe(df_counties, directory, filename, file_format)
+
+        filename = "state_divi"
+        filename = gd.append_filename(filename, impute_dates, moving_average)
+        gd.write_dataframe(df_states, directory, filename, file_format)
 
-    return (df_raw, df_counties, df_states, df_ger)
+        filename = "germany_divi"
+        filename = gd.append_filename(filename, impute_dates, moving_average)
+        gd.write_dataframe(df_ger, directory, filename, file_format)
 
+    return df_counties, df_states, df_ger
```

Comment:
Please be consistent if the DataFrames are returned or not dependant on `to_dataset`. Also how they're returned. In getCaseData you used a dict. Here they're seperately returned.

Reply:
I think I had to otherwise the unit test would fail.

Reply:
I can change the test and account for the fact that sometimes no data is returned

Reply:
I have now created and returned a dictionary for consistency.